src/colornames.csv

```diff
@@ -19372,6 +19434,7 @@ Pavestone,#c9c4ba,
 Pavilion,#bebf84,
 Pavilion Beige,#c5b6a4,
 Pavilion Peach,#df9c45,
+Pavilion Tan,#b49483,

```

Comment:
I'd like to remove that because it assumes that the person getting this Tan is white.